docs/deploying-airbyte/on-aws-ecs.md

```diff
@@ -6,5 +6,7 @@ We do not currently support deployment on ECS.
 
 :::
 
-The current iteration is not compatible with ECS. Airbyte currently relies on docker containers being able to create other docker containers. ECS does not permit containers to do this. We will be revising this strategy soon, so that we can be compatible with ECS and other container services.
-
+The current iteration is not compatible with ECS. 
```

Comment:
markdown doesn't care about single carriage returns and I think this read easier for people in shell and text editors